pyha_analyzer/train.py

```diff
@@ -227,6 +226,7 @@ def inference_valid(self):
         domain_shift = np.abs(self.valid_map - infer_map)
         wandb.log({
             "valid/domain_shift_diff": domain_shift,
+            "valid/inferance_map": infer_map,
```

Comment:
typo

Reply:
Intentional typo because sean spelled it wrong somewhere else and I don't want a name conflict